Vagrantfile

```diff
@@ -4,31 +4,27 @@
 require 'yaml'
 AWS = YAML.load_file 'aws.yml'
 
+if Vagrant::VERSION < '2.2.4'
```

Comment:
Using the following will be shorter

Vagrant.require_version ">= 2.2.4"

Reply:
done